### PR TITLE
Add icon font awesome

### PR DESCRIPTION
**Pull request by PhilippeBarre**
Note I have replace "website" by "globe" in src\contributors_details.json because website is not a font-awesome icon

Issue : Add icon to the contributors page #14

### CODE REVIEW

**Comment by PhilippeBarre** [comment]
Humm the build failed because of the following import "unused"
import { faGlobe, faPhone } from '@fortawesome/fontawesome-free-solid'
But this import is needed to load icon

File: src/components/contributors.js [hunk 1]
```diff
@@ -17,7 +19,7 @@ let mapDetail = (detail) => {
               target="_blank"
               rel="noreferrer noopener"
             >
-              Profile
+              <FontAwesomeIcon icon={contact.icon} />
```
Comment (niranjanhegde144) [inline src/components/contributors.js:22]: Need to add a condition to display the icon. Based on the icon value from the JSON file you need to choose the font awesome icon to be displayed.

Comment (PhilippeBarre) [reply src/components/contributors.js:22]: No need conditions, this soluce work : 
image

File: src/contributors_details.json [hunk 2]
```diff
@@ -33,7 +33,7 @@
     "contact": [
       {
         "id": "contact1",
-        "icon": "website",
+        "icon": "globe",
         "link": "https://niranjanhegde144.github.io/portfolio/"
       },
       {
```
Comment (niranjanhegde144) [inline src/contributors_details.json:39]: It's better to use the font awesome desktop icon in place of the globe icon.

Comment (PhilippeBarre) [reply src/contributors_details.json:39]: Okey I will change it

File: src/contributors_details.json [hunk 3]
```diff
@@ -33,7 +33,7 @@
     "contact": [
       {
         "id": "contact1",
-        "icon": "website",
+        "icon": "globe",
         "link": "https://niranjanhegde144.github.io/portfolio/"
       },
```
Comment (niranjanhegde144) [inline src/contributors_details.json]: I think you can use the font awesome desktop icon in place of the globe icon. So you can change the value globe to desktop.

**Review by niranjanhegde144 — CHANGES_REQUESTED** [review]

**Comment by niranjanhegde144** [comment]
> Humm the build failed because of the following import "unused"
> import { faGlobe, faPhone } from '@fortawesome/fontawesome-free-solid'
> But this import is needed to load icon

The error you are recieving

File: src/contributors_details.json (hunk 3 above)
Comment (PhilippeBarre) [reply src/contributors_details.json]: Yes it is already done ;)

Comment (PhilippeBarre) [reply src/contributors_details.json]: image

**Comment by niranjanhegde144** [comment]
> Humm the build failed because of the following import "unused"
> import { faGlobe, faPhone } from '@fortawesome/fontawesome-free-solid'
> But this import is needed to load icon

That's because the icon value is being just replaced as phone, or globe. You want it to be faPhone and faGlobe. So you need to use icon = {"fa" + contact.icon.charAt(0).toUpperCase() + contact.icon.slice(1);

**Comment by PhilippeBarre** [comment]
> > Humm the build failed because of the following import "unused"
> > import { faGlobe, faPhone } from '@fortawesome/fontawesome-free-solid'
> > But this import is needed to load icon
> 
> That's because the icon value is being just replaced as phone, or globe. You want it to be faPhone and faGlobe. So you need to use icon = {"fa" + contact.icon.charAt(0).toUpperCase() + contact.icon.slice(1);

I tested this, this work too but the warning "defined but never used" is still there

**Comment by PhilippeBarre** [comment]
The preview is okey : https://5f78a3f42291a000072455c0--sce-tech-club.netlify.app/contributors ?

**Comment by niranjanhegde144** [comment]
I've added the tag and labeled the pull request as hacktoberfest-accepted. If it's still giving a warning then you might have to use an if condition.

**Comment by PhilippeBarre** [comment]
> If it's still giving a warning then you might have to use an if condition for the icon rendering.

Yep I will do it

**Comment by niranjanhegde144** [comment]
I'll be merging the changes. Thank you for the help.

**Pull request merged**